utils/src/main/java/com/cloud/utils/AutoCloseableUtil.java

```diff
@@ -16,10 +16,11 @@
 // under the License.
 package com.cloud.utils;
 
-import org.apache.log4j.Logger;
+import org.apache.cloudstack.utils.log.LogFactory;
+import org.apache.cloudstack.utils.log.Logger;
 
 public class AutoCloseableUtil {
-    private final static Logger s_logger = Logger.getLogger(AutoCloseableUtil.class);
+    private final static Logger s_logger = LogFactory.getLogger(AutoCloseableUtil.class);
```

Comment:
I guess we'll need to refactor codebase wide to use LogFactory.

Reply:
I would regard that out of scope for this PoC, but part of a series of follow-up PRs.